diff --git a/node_modules/fetch-retry/index.d.ts b/node_modules/fetch-retry/index.d.ts
index f120ec1..96500a7 100644
--- a/node_modules/fetch-retry/index.d.ts
+++ b/node_modules/fetch-retry/index.d.ts
@@ -22,5 +22,5 @@ declare module 'fetch-retry' {
   }
 
   function fetchBuilder(fetch: typeof _fetch, defaults?: object): ((input: RequestInfo, init?: RequestInitWithRetry) => Promise<Response>);
-  export = fetchBuilder;
+  export default fetchBuilder;
 }
